hoomd/conftest.py

```diff
@@ -132,3 +136,1518 @@ def pytest_sessionfinish(session, exitstatus):
 
     if exitstatus != 0 and hoomd._hoomd.is_MPI_available():
         atexit.register(abort, exitstatus)
+
+
+@pytest.fixture(scope="session")
```

Comment:
These should be in `hoomd/hpmc/conftest.py`. Also, they should only have a function scope.